### PR TITLE
Boilerplate spec and various Jakarta EE 8 related updates

### PR DESCRIPTION
**Pull request by m0mus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 [![Build Status](https://travis-ci.org/eclipse-ee4j/jsonp.svg?branch=master)](https://travis-ci.org/eclipse-ee4j/jsonp)
 
-# JSON Processing (JSON-P)
+# Jakarta JSON Processing (JSON-P)
 
-This projects contains API and Reference Implementation (RI) of JSR-374. 
-The Java API for JSON Processing provides portable APIs to parse, generate, transform, and query JSON documents.
+The Java API for JSON Processing provides portable APIs to parse, generate, transform, and query JSON documents. This projects contains Jakarta JSON Processing specification, API and a compatible implementation.
```
Comment (lukasj) [inline README.md]: either 'this project' or 'these projects' but not 'this projects'

Comment (m0mus) [reply README.md]: ooops...

Comment (m0mus) [reply README.md]: Fixed.

File: spec/pom.xml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                        <id>enforce-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireJavaVersion>
+                                    <version>[1.8.0,1.9.0)</version>
+                                    <message>You need JDK8 or lower</message>
+                                </requireJavaVersion>
```
Comment (lukasj) [inline spec/pom.xml]: I'm changing this to:
```
<version>[1.8.0,)</version>
<message>You need JDK8 or newer</message>
```

File: spec/pom.xml [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+                <executions>
+                    <execution>
+                        <id>asciidoc-to-html</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>process-asciidoc</goal>
+                        </goals>
+                        <configuration>
+                            <backend>html5</backend>
+                            <outputFile>${project.build.directory}/generated-docs/jsonp-spec-${project.version}.html</outputFile>
```
Comment (lukasj) [inline spec/pom.xml]: I'd prefer the spec to be called 'jakarta.json-spec-...'

File: spec/pom.xml (hunk 2 above)
Comment (m0mus) [reply spec/pom.xml]: Agree. I thought it doesn't work on JDK > 8, but it does, I checked.

File: spec/pom.xml (hunk 3 above)
Comment (m0mus) [reply spec/pom.xml]: I would rather keep it as it is now.

**Pull request merged**